### PR TITLE
Auxiliary class enhancements

### PR DESCRIPTION
**Pull request by fabgio**
Add requestTurnOn RequestTurnOff methods to Auxiliary class

### CODE REVIEW

File: src/main/java/org/openwebnet4j/message/Auxiliary.java [hunk 1]
```diff
@@ -54,12 +58,67 @@ public Integer value() {
...
+     *
+     * @param where WHERE string
+     * @return message
+     */
+    public static Auxiliary requestTurnOn (String where){
+        return new Auxiliary(format(FORMAT_REQUEST, WHO, WhatAuxiliary.ON.value,where));
+    }
+
+    /**
+     * OpenWebNet message request to turn auxiliary ON <code>*9*0*WHERE##</code>.
```
Comment (mvalla) [inline src/main/java/org/openwebnet4j/message/Auxiliary.java:78]:
```suggestion
     * OpenWebNet message request to turn auxiliary OFF <code>*9*0*WHERE##</code>.
```

**Review by mvalla — CHANGES_REQUESTED** [review]
just a small correction to the javadoc for requestTurnOff()

**Comment by fabgio** [comment]
Ok for the coding  issues of the binding.
As far as  drafting the first version for other people to test it, if I am right I   would fork your fork  of openhab-addons and send PRs there, isn’t it? So you may  review the code as well. Finally send  PR to openhab-addons

**Comment by mvalla** [comment]
No, you should directly fork openhab-addons. If you want I can look at it before sharing it for others to test. I can use your fork to compile and test myself.

**Comment by mvalla** [comment]
And first of all you should open a new issue on openhab-addons

**Comment by fabgio** [comment]
Hi, I have been thinking about  the integration of  AUX feature  in the openwebnet binding   but  I suppose  I have to wait for    version 0.7 of openwebnet4j  to be released first; any schedule for that?
Thanks
Giovanni

**Comment by mvalla** [comment]
If for the moment you do not have other new changes for the lib which are needed for AUX, I can publish a 0.7.0 version this evening

**Comment by fabgio** [comment]
Ok I have just completed some tests, I have no new changes to make! As far as I am concerned you can publish the 0.7.0 version of the openwebnet4j library

**Comment by fabgio** [comment]
Any news?

**Comment by mvalla** [comment]
0.7.0 just [published on Maven Central.](https://repo1.maven.org/maven2/io/github/openwebnet4j/openwebnet4j/0.7.0/) 
Remember to change 0.7.0-SNAPSHOT to 0.7.0 in your pom.xml

**Comment by fabgio** [comment]
Great! Thanks!

**Comment by fabgio** [comment]
I completed   the implementation of the AUX feature, built it smoothly and successfully  performed  unit  tests for the binding. I did  code analysis as well. Still, it ended up with  a warning regarding OpenWebnetEnergyHandler class [136,13] "potential null pointer access" on a variable called "ns"; btw, I watched the code and spotted that  it was  correctly checked for non null. I finally read the coding guide of openhab and everything seem to comply with it. I am about the send the PR Any suggestion?
Would you look at it? 
Thanks

**Comment by mvalla** [comment]
Sure I will have a look: just need to find the time. I suggest you send the PR anyway as it takes time to get reviewed. I suggest also you anticipate on a community thread the AUX version (or maybe use the new OH3 marketplace feature) so that some user with AUX configuration can test on a real plant (I have only a simulation of AUX)

**Pull request merged**